atom/browser/api/atom_api_web_contents.h

```diff
@@ -139,6 +139,7 @@ class WebContents : public mate::TrackableObject<WebContents>,
   // Methods for creating <webview>.
   void SetSize(const SetSizeParams& params);
   bool IsGuest() const;
+  bool IsRemote() const;
```

Comment:
There is already a `GetType` method.

For the purpose of performance, we can make the `GetType` method return the `Type` enum, while adding a converter to convert it to V8 value.